spec/models/employment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Employment, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Did you mean to leave this as pending?

Reply:
Yes, just a reminder to add tests for next screen